### ISSUE TITLE
Accepting MRs

### ISSUE THREAD

**Issue by sangimed** [gmessner/gitlab4j-api#54]
Hello,

Is it possible to [accept merge requests](https://docs.gitlab.com/ee/api/merge_requests.html#accept-mr) using the current version (4.4.1) ? I didn't find any method that seems to do that on the [MergeRequestApi.java](https://github.com/gmessner/gitlab4j-api/blob/master/src/main/java/org/gitlab4j/api/MergeRequestApi.java).

Thank you in advance.

**Comment by gmessner** [comment]
@jsimomaa 
Accepting merge requests is not currently supported.  I will be adding support for this and releasing in the next day or so.

**Comment by gmessner** [comment]
The following has been added to MergeRequestApi and is available in release 4.4.3:
```java
public MergeRequest acceptMergeRequest(Integer projectId, Integer mergeRequestId, 
           String mergeCommitMessage, Boolean shouldRemoveSourceBranch,
           Boolean mergeWhenPipelineSucceeds)
```
